### PR TITLE
BACKLOG-16510: Add GraphqlQueryTest in Cypress

### PR DESCRIPTION
**Pull request by gflores-jahia**
## JIRA

https://jira.jahia.org/browse/BACKLOG-16510

## Description

Add/migrate [GraphqlQueryTest](https://github.com/Jahia/graphql-core/blob/master/graphql-test/src/main/java/org/jahia/test/graphql/GraphQLQueryTest.java) from Java test suite to cypress integration tests

### CODE REVIEW

**Review by tdraier — DISMISSED** [review]

File: tests/cypress/integration/api/jcr/graphqlQueryTest.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import gql from 'graphql-tag'
+
+describe('GraphQL Query Test', () => {
+
+    /* Setup */
+
+    before('Create query test nodes', () => {
+        cy.apollo({
+            mutation: gql`mutation {
+                jcr {
```
Comment (tdraier) [inline tests/cypress/integration/api/jcr/graphqlQueryTest.ts]: use addNode.graphql when available

File: tests/cypress/integration/api/jcr/graphqlQueryTest.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                    }
+                }
+            `,
+            variables: { query, queryLanguage },
+            errorPolicy
+        })
+    }
+
+    function validateError(result, errorMessage) {
+        expect(Array.isArray(result.errors)).to.be.true;
```
Comment (tdraier) [inline tests/cypress/integration/api/jcr/graphqlQueryTest.ts]: also use method from validateErrors.ts when available

Comment (gflores-jahia) [reply tests/cypress/integration/api/jcr/graphqlQueryTest.ts]: synced up and resolved with last commit

File: tests/cypress/integration/api/jcr/graphqlQueryTest.ts (hunk 1 above)
Comment (gflores-jahia) [reply tests/cypress/integration/api/jcr/graphqlQueryTest.ts]: migrated to reuse addNode batch query

**Review by tdraier — APPROVED** [review]

**Pull request merged**